.claude/skills/roadmap-sync/sync.sh

```
@@ -0,0 +1,121 @@
+  items_json="$(gh project item-list "$PROJECT" --owner "$OWNER" --format json -L "$ITEM_LIMIT")"
+  if [ "$(jq '.totalCount' <<<"$items_json")" -gt "$(jq '.items | length' <<<"$items_json")" ]; then
+    say "project has more than $ITEM_LIMIT items; raise ITEM_LIMIT before syncing"
+    exit 1
+  fi
+}
+load_items
+
+while IFS=$'\t' read -r id title; do
+  say "draft \"$title\": converting to an issue"
+  run gh api graphql -F itemId="$id" -F repositoryId="$REPO_ID" -F title="$title" \
+    -f query='mutation($itemId:ID!,$repositoryId:ID!,$title:String!){convertProjectV2DraftIssueItemToIssue(input:{itemId:$itemId,repositoryId:$repositoryId,title:$title}){item{id}}}'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)